l10n/en/products/vpn/shared.ftl

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@maxxcrawford this is what I was thinking for shared strings, let me know your thoughts! 

Currently this includes things like:

- CTA button text.
- Available countries where VPN is offered.
- Supported platform info.
- Pricing info.
- Product features.
- Privacy and copyright text.

Strings here will almost certainly change in the next week or so, but hopefully we can align on what will be used here between pages.

Also, if there are any other strings you think might be useful here for the SEO pages then let me know! Likewise, if there are common elements or sections of the landing page that could be shared on other SEO pages, then I can look to make those includes at the template level.

Reply:
This looks/works great. I'm not sure about shared components yet. I keep thinking the pricing box and cta may be one component.

Reply:
Okay – I checked out the copy doc for the other pages and found only a few strings that were repeated (without some variation): 

- What you’ll get with Mozilla VPN
- Protection for up to 5 devices
- Access to servers in 30+ countries
- Device-level encryption
- No bandwidth restrictions
- No online activity logs now or ever
- Connect up to 5 devices
- 30-day money-back guarantee

Reply:
Discalimer: Some of these lines are not finalized, but they are used in each section. (Up to five in use on each page)

Reply:
@maxxcrawford most of the strings you mention here in relation to VPN features are almost (but not quite) the same as the landing page strings. Perhaps this is something we could try and standardize on once we're closer to final copy 👍